test/test_latex.py

```diff
@@ -0,0 +1,12 @@
+"""
+Tests for uqcsbot.scripts.cat
+"""
+from .conftest import MockUQCSBot
+from .helpers import generate_message_object
+
+
+def test_latex(uqcsbot: MockUQCSBot):
```

Comment:
this test needs updating to the new test framework (also, can you adopt the new styling too)